### ISSUE TITLE
File Manager: Differentiate directory names

### ISSUE THREAD

**Issue by qooob** [webmin/webmin#542]
Jamie, I think I agree with Joe's [comment](https://github.com/qooob/authentic-theme/issues/731#issuecomment-298213501) and it's high time to rename `filemin` into `file-manager` to avoid confusion. I suppose user's ACLs should be updated accordingly, if it's not very difficult to do.

**Comment by jcameron** [comment]
To avoid clashes with the newer filemin module?

**Comment by qooob** [comment]
Yes, some people for unknown (to me) reasons install 2, incompatible version of it. We should let them do it with out overwriting stock version.

**Comment by jcameron** [comment]
OK - I need to figure out how to handle a rename without impacting existing users.

**Comment by qooob** [comment]
Are we doing it for 1.840?

**Comment by swelljoe** [comment]
There haven't been any updates in Real-Gecko's version of Filemin in 8+ years, so I guess this issue is moot.
